app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.errors import VaccineError
+from app.errors import NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: str) -> str:
```

Comment:
incorrect type annotation

Reply:
it's correct because the function returns string

Reply:
It's incorrect because `cafe` is not string

Reply:
fixed